dashboard/apps/prototype/models.py

```diff
@@ -1,3 +1,87 @@
+# -*- coding: utf-8 -*-
 from django.db import models
 
-# Create your models here.
+
+class Person(models.Model):
+    float_id = models.CharField(max_length=64, unique=True)
+    name = models.CharField(max_length=64)
+    email = models.EmailField()
+    roles = models.ManyToManyField('Role', through='PersonRole')
+
+    def __str__(self):
+        return self.name
+
+
+class Role(models.Model):
+    name = models.CharField(max_length=32)
+    people = models.ManyToManyField('Person', through='PersonRole')
+    default_rate = models.DecimalField(max_digits=5, decimal_places=2)
+    is_contractor = models.BooleanField()
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        unique_together = ('name', 'is_contractor')
+
+
+class Rate(models.Model):
+    amount = models.DecimalField(max_digits=5, decimal_places=2)
+    person = models.ForeignKey('Person', related_name='rates')
+    start_date = models.DateTimeField()
+    end_date = models.DateTimeField()
```

Comment:
maybe set null=True on end_date as an end_date won't be know

you could even get rid of end_date all together as the end date will be the start date of the later entry